lib/rules/no-useless-escape.js

```diff
@@ -9,6 +9,19 @@
 // Rule Definition
 //------------------------------------------------------------------------------
 
+/**
+* Returns the union of two sets.
+* @param {Set} setA The first set
+* @param {Set} setB The second set
+* @returns {Set} The union of the two sets
+*/
+function union(setA, setB) {
```

Comment:
This is so cool.. I never knew abt `yield*`.
Had to read up on couple of things to understand this but so awesome. 💯